diff --git a/utils/general.py b/utils/general.py
index e95b074..8c95884 100644
--- a/utils/general.py
+++ b/utils/general.py
@@ -35,7 +35,7 @@ import pkg_resources as pkg
 import torch
 import torchvision
 import yaml
-from ultralytics.yolo.utils.checks import check_requirements
+from ultralytics.utils.checks import check_requirements
 
 from utils import TryExcept, emojis
 from utils.downloads import curl_download, gsutil_getsize
